DirectProgramming/DPC++FPGA/ReferenceDesigns/mvdr_beamforming/README.md

```diff
@@ -0,0 +1,216 @@
+# MVDR Beamforming
+This reference design demonstrates FPGA IO streaming in oneAPI. 
+
+***Documentation***:  
+* [DPC++ FPGA Code Samples Guide](https://software.intel.com/content/www/us/en/develop/articles/explore-dpcpp-through-intel-fpga-code-samples.html) helps you to navigate the samples and build your knowledge of DPC++ for FPGA. <br>
+* [oneAPI DPC++ FPGA Optimization Guide](https://software.intel.com/content/www/us/en/develop/documentation/oneapi-fpga-optimization-guide) is the reference manual for targeting FPGAs through DPC++. <br>
+* [oneAPI Programming Guide](https://software.intel.com/en-us/oneapi-programming-guide) is a general resource for target-independent DPC++ programming. 
+ 
+| Optimized for                     | Description
+---                                 |---
+| OS                                | Linux* Ubuntu* 18.04; Windows* 10
+| Hardware                          | Intel&reg; Programmable Acceleration Card (PAC) with Intel Arria&reg; 10 GX FPGA <br> Intel&reg; FPGA Programmable Acceleration Card (PAC) D5005 (with Intel Stratix&reg; 10 SX) <br> Intel Xeon&reg; CPU E5-1650 v2 @ 3.50GHz (host machine)
+| Software                          | Intel&reg; oneAPI DPC++ Compiler <br> Intel&reg; FPGA Add-On for oneAPI Base Toolkit
+| What you will learn               | How to perform IO streaming in oneAPI using the IO pipes SYCL extension.
+| Time to complete                  | 1 hour
+
+## Purpose
+The purpose of this tutorial is to implement a high-performance streaming IO design using oneAPI. In this reference design, we implement an MVDR-beamforming algorithm using oneAPI. More details on the algorithm can be found in the [Additional Design Information](#additional-design-information) Section.
+
+## Key Implementation Details
+This reference design code sample leverages concepts that are discussed in the following FPGA tutorials: 
+* **IO Streaming** (io_streaming)
+* **Explicit Pipelining with `fpga_reg`** (fpga_register)
+* **Loop `ivdep` Attribute** (loop_ivdep)
+* **Triangular Loop Optimization** (triangular_loop)
+* **Unrolling Loops** (loop_unroll)
+* **Pipe Arrays** (pipe_array)
+
+Notably, we strongly suggest reviewing the **IO Streaming** code sample as this reference design is a direct extension of the concepts it describes.
```

Comment:
I'm really surprised not to see a brief discussion of "faking" IO streaming here! I don't think that a reference back to the IO streaming tutorial is sufficient -- I think it's worth a brief blurb to explain that the data in this design is actually coming from the host, that this is a handy development technique, and to see IO streaming for more details.

Update -- I found this at the bottom of the README with the design details. I suggest moving that here as it's a pretty key piece of information for anyone reading (or thinking of running) this sample.